cmd/run.go

```diff
@@ -151,17 +153,20 @@ func (c *cmdRun) run(cmd *cobra.Command, args []string) error {
 
 	// We do this here so we can get any output URLs below.
 	initBar.Modify(pb.WithConstProgress(0, "Starting outputs"))
-	outputManager := output.NewManager(outputs, logger, func(err error) {
-		if err != nil {
-			logger.WithError(err).Error("Received error to stop from output")
-		}
-		runCancel()
-	})
-	err = outputManager.StartOutputs()
+	// TODO: re-enable the code below
+	/*
+		outputManager := output.NewManager(outputs, logger, func(err error) {
+			if err != nil {
+				logger.WithError(err).Error("Received error to stop from output")
+			}
+			runCancel()
+		})
+	*/
```

Comment:
When would you re-enable it? I guess when the engine will be totally removed but not sure, so can you make it clear, please? :pray: